### PR TITLE
Graph lint error patch

### PR DESCRIPTION
**Pull request by jjsjann123**
- [x] extend buildShapeExpression for squeeze_copy/unsqueeze_copy ops.
- [x] patching broadcastSizes insertion point for buildShapeExpression to avoid graph::copy() linter assert.
- [x] adding tests
- [x] supports no-op squeeze (squeezing on dimension that's not size-1)

TODO (in follow up PRs):
1. extend buildShapeExpression to `view_copy` and `reshape_copy` as well
2. refactor broadcastSizesExpression to allow graceful failure instead of hard assert

### CODE REVIEW

**Comment by jjsjann123** [comment]
Ready for review now.

Should have split it into a few PRs but they are all relatively small so there we go.

File: torch/csrc/jit/codegen/cuda/interface.cpp [hunk 1]
```diff
@@ -652,6 +652,68 @@ RegisterOperators reg_unsqueeze_copy({
         },
         aliasAnalysisFromSchema()),
 });
+
```
Comment (rdspring1) [inline torch/csrc/jit/codegen/cuda/interface.cpp:655]: Maybe we should move the custom operators to a separate file in the future.

**Review by rdspring1 — APPROVED** [review]
LGTM

**Comment by jjsjann123** [comment]
Tests passing I'm merging this after CI.

**Pull request merged**